bigquery/samples/tests/conftest.py

```diff
@@ -65,6 +65,15 @@ def dataset_id(client):
     client.delete_dataset(dataset, delete_contents=True, not_found_ok=True)
 
 
+@pytest.fixture
+def dataset_with_labels_id(client, dataset_id):
```

Comment:
Optional: Rather than make this fixture, we could combine all the "labels" tests into a common test file. See the models samples for an example.

Reply:
I decided to follow your advice and combined all `dataset_label` tests into one.